tools/trace_example.txt

```diff
@@ -435,13 +464,9 @@ trace -I 'kernel/sched/sched.h' \
         in kernel/sched/sched.h which is in kernel source tree and not in kernel-devel
         package.  So this command needs to run at the kernel source tree root directory
         so that the added header file can be found by the compiler.
-trace -I 'net/sock.h' \\
+trace -I 'net/sock.h' \
       'udpv6_sendmsg(struct sock *sk) (sk->sk_dport == 13568)'
         Trace udpv6 sendmsg calls only if socket's destination port is equal
         to 53 (DNS; 13568 in big endian order)
 trace -I 'linux/fs_struct.h' 'mntns_install "users = %d", $task->fs->users'
         Trace the number of users accessing the file system of the current task
-trace -s /lib/x86_64-linux-gnu/libc.so.6,/bin/ping 'p:c:inet_pton' -U
```

Comment:
any reason why you want to remove this example?

Reply:
I pasted the full text of 'trace -h', and the text is a little different from trace_example.txt.
So I suppose there are two options:
1, keep usage information in trace_example.txt aligned with 'trace -h'
2, revert the these minor changes, only append new option '-A' part

Which one do you prefer?

Reply:
Let us keep it aligned. So that means to add the above
   trace -s /lib/x86_64-linux-gnu/libc.so.6,/bin/ping 'p:c:inet_pton' -U
to trace.py,
and fix the following in trace.py,
  Trace the open syscall and print the filename. being opened @user is

Reply:
Sure, force-pushed.